windows/check_quiesce_snapshot/check_quiesce_snapshot.yml

```diff
@@ -16,61 +17,30 @@
           vars:
             skip_test_no_vmtools: true
 
-        - name: "Set fact of the remote path in guest OS"
-          ansible.builtin.set_fact:
-            quiesce_scripts_freeze_thaw_path: "C:\\Windows\\"
-            quiesce_scripts_tools_path: "C:\\Program Files\\VMware\\VMware Tools\\backupScripts.d\\"
-            quiesce_log_file_path: "C:\\vmbackup.log"
-
-        - name: "Check disk.EnableUUID in VMX file"
+        - name: "Check disk.EnableUUID in VMX file for Windows Server"
           include_tasks: check_vmx_disk_enable_uuid.yml
           when: guest_os_product_type == "server"
 
-        - name: "Enable vss logging"
-          include_tasks: ../utils/win_enable_vss_log.yml
-          vars:
-            win_vmbackup_log_path: "{{ quiesce_log_file_path }}"
-          when:
-            - vmtools_version is defined and vmtools_version
-            - vmtools_version is version('11.0.0', '>=')
-
-        - name: "Get Windows guest OS time"
-          include_tasks: ../utils/win_get_time.yml
-
-        - name: "Set fact of the time before taking snapshot"
-          ansible.builtin.set_fact:
-            before_snapshot_time: "{{ time_variable }}"
-
-        - name: "Set fact of the quiesce snapshot name"
-          ansible.builtin.set_fact:
-            qs_snapshot_name: "quiesce_{{ current_test_timestamp }}"
-
-        - name: "Copy custom scripts to guest OS"
-          include_tasks: copy_custom_scripts_to_guest.yml
-
-        - name: "Get VMware Snapshot Provider service status in guest OS"
-          include_tasks: ../utils/win_get_service_status.yml
-          vars:
-            win_service_name: "vmvss"
-
-        - name: "Set fact of collecting quiesce log file"
-          ansible.builtin.set_fact:
-            collect_vss_logs: true
+        - name: "Preparation for this test case"
+          include_tasks: prepare_quiesce_snapshot.yml
 
         - name: "Take quiesce snapshot of VM"
           include_tasks: ../../common/vm_take_snapshot.yml
           vars:
-            snapshot_name: "{{ qs_snapshot_name }}"
+            snapshot_name: "{{ quiesce_snapshot_name }}"
             is_quiesce: true
             dump_memory: false
 
         - name: "Check custom scripts executed"
           include_tasks: check_custom_scripts_executed.yml
 
+        - name: "Check quiescing type"
```

Comment:
Is it better to add some description that there are two types of quiese snapshot? Filesystem only generates one backup.xml file, APP will generate more then one file.

Reply:
It can be seen in the below tasks. 
```
- name: "Check filesystem consistent quiescing taken"
  ansible.builtin.assert:
    that:
      - vm_vss_manifests_files.files is defined
      - vm_vss_manifests_files.files | length == 1
      - vm_vss_manifests_files.files[0] == "backup.xml"
    fail_msg: >
      One file 'backup.xml' is expected in {{ vm_vss_manifests_new[0] | basename }} when
      taking filesystem consistent quiesce snapshot, while got files: {{ vm_vss_manifests_files.files | default([]) }}.
  when: not winsrv_app_quiesce_support

- name: "Check application consistent quiescing taken for Windows Server"
  ansible.builtin.assert:
    that:
      - vm_vss_manifests_files.files is defined
      - vm_vss_manifests_files.files | length > 1
    fail_msg: >
      More than one files are expected in {{ vm_vss_manifests_new[0] | basename }} when
      taking application consistent quiesce snapshot, while got file: {{ vm_vss_manifests_files.files | default([]) }}.
  when: winsrv_app_quiesce_support
```

Reply:
I know. I mean adding some description either in test case description part or before the task. It can make it easier to understand the logic in check_quiesec_type.yml. 
Whatever, it depends on you.

Reply:
The logic is easy to understand as above pasted task samples and it's explained clearly in the fail messages. Anything you don't understand please let me know.

Reply:
I'm not saying the pasted tasks are not clear. I just believe adding a brief description before looking into the included yml file will make user easier to have an overview over the quiese snapshot types. 

It's a better have thing, not required thing. It's totally up to you.

Reply:
I'm saying above tasks are clear to explain if other users like you don't understand the logic of the included task, no need to add the description before including task file, and thanks for your comments.